src/server.c

```diff
@@ -3324,6 +3323,13 @@ void initServer(void) {
     scriptingInit(1);
     slowlogInit();
     latencyMonitorInit();
+    
+    /* Initialize ACL default password if it exists */
+    if (server.requirepass) {
+        sds aclop = sdscatlen(sdsnew(">"), server.requirepass, sdslen(server.requirepass));
+        ACLSetUser(DefaultUser,aclop,sdslen(aclop));
+        sdsfree(aclop); 
+    }
```

Comment:
maybe it would be safer for future changes and cleaner to just call `updateRequirePass` here?

Reply:
Sure